### DISCUSSION TITLE
Cube https://environment.ld.admin.ch/foen/nfi/49-19-44/cube/1 has disappeared ?

### DISCUSSION THREAD

**Discussion by ptbrowne** [visualize-admin/visualization-tool#1007] [General]
Hi @rdataflow or @fabiancretton,

It seems like the cube https://environment.ld.admin.ch/foen/nfi/49-19-44/cube/1 has disappeared. It was used previously in some tests. Do you know what happened to it ?

**Comment by FabianCretton** [comment]
Hi @ptbrowne, I do not know this cube and do not know what happened.
On which environment was it ?
I guess Thomas will have the info, he should be back next week

**Comment by Rdataflow** [comment]
@kronmar oups, can you inform @ptbrowne on the transition plans and what to monitor next

@ptbrowne is it fine to move on with the tests ? (or would you need a restored version of this legacy cube?)
